rollup/pipeline/pipeline.go

```diff
@@ -103,22 +112,20 @@ func (p *Pipeline) WithBeforeTxHook(beforeTxHook func()) *Pipeline {
 func (p *Pipeline) Start(deadline time.Time) error {
 	p.start = time.Now()
 	p.txnQueue = make(chan *types.Transaction)
-	applyStageRespCh, candidateCh, err := p.traceAndApplyStage(p.txnQueue)
+	applyStageRespCh, applyDownstreamCh, err := p.traceAndApplyStage(p.txnQueue)
```

Comment:
This is a bit confusing, though it's not a bad name. How about using `applyToEncodeCh` and `encodeToCccCh`?

Reply:
Sounds good.

Reply:
done in https://github.com/scroll-tech/go-ethereum/pull/814/commits/278f7a94fa08dcf0df88d4c5f636e3cfb5234dd6